services/download/pom.xml

```diff
@@ -20,6 +23,11 @@
       <artifactId>postgresql</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.commons</groupId>
+          <artifactId>commons-collections4</artifactId>
+          <version>${commons-collections4.version}</version>
+      </dependency>
```

Comment:
I would like to not depend on more and more other libraries...

Reply:
Removed dependency.